opent2t.p.hub/com.wink.hub/js/thingTranslator.js

```diff
@@ -284,14 +322,19 @@ class Translator {
     /**
      * Internal helper method which makes the actual request to the wink service
      */
-    _makeRequest(path, method, content) {
+    _makeRequest(path, method, content, includeBearerHeader) {
+       
+        var addBearerHeader = (typeof includeBearerHeader !== 'undefined') ?  includeBearerHeader : true;
```

Comment:
since this is a stop gap at the moment ntil we figure out ohw to upgrade to latest JS that supports default params, keeping it as is

---
In reply to: [89005342](https://github.com/openT2T/translators/pull/126#discussion_r89005342) [](ancestors = 89005342,89002893,88971888)